src/transform/FooterMenu.js

```diff
@@ -0,0 +1,215 @@
+import './FooterMenu.css'
+
+/**
+ * @typedef {function} FooterMenuItemPayloadExtractor
+ * @param {!Document} document
+ * @return {!string[]} Important - should return empty array if no payload strings.
+ */
+
+/**
+  * @typedef {function} FooterMenuItemClickCallback
+  * @param  {!string[]} payload Important - should return empty array if no payload strings.
+  * @return {void}
+  */
+
+/**
+ * Extracts array of no-html page issues strings from document.
+ * @type {FooterMenuItemPayloadExtractor}
+ */
+const pageIssuesStringsArray = document => {
```

Comment:
See the `FooterMenuItemPayloadExtractor` typedef for the details. I think I did this correctly according to the `Type definitions` box in the [docs](http://usejsdoc.org/tags-type.html). Lemme know if not :)